sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BulkExecutor.java

```diff
@@ -866,7 +866,18 @@ private void completeAllSinks() {
         if (completeEmitResult == Sinks.EmitResult.OK) {
             logger.debug("Main sink completed, Context: {}", this.operationContextText);
         } else {
-            logger.info("Main sink completion failed. EmitResult: {}, Context: {}", completeEmitResult, this.operationContextText);
+            if (completeEmitResult == Sinks.EmitResult.FAIL_CANCELLED ||
```

Comment:
Not part of this PR, but just curious to see that logger on line 862 - is info. Should we also change it to debug?

Reply:
No - 862 being info is helpful - because it helps debugging in scenarios where some ingestion tasks take a long time (hang vs. long running task)